server/src/main/java/cn/nukkit/server/level/noise/Noise.java

```diff
@@ -0,0 +1,135 @@
+package cn.nukkit.server.level.noise;
+
+public abstract class Noise {
```

Comment:
these appear to be the normal nukkit noise classes, but they're buggy and slow (source: i worked with them for a good 2 weeks non-stop). use the notchian ones, they're on master

Reply:
And I was intending the use a noise library instead of having the code inside Nukkit

Reply:
and i already started!

Reply:
https://github.com/PorkLib/noise

Reply:
ah, i'll remove the noise classes then